src/site/stages/build/process-cms-exports/schemas/transformed/node-press_release.js

```diff
@@ -0,0 +1,81 @@
+module.exports = {
+  type: 'object',
+  properties: {
+    contentModelType: { enum: ['node-press_release'] },
+    entityType: { enum: ['node'] },
+    entityBundle: { enum: ['press_release'] },
+    title: { type: 'string' },
+    entityMetatags: {
+      type: 'array',
+      items: {
+        type: 'object',
+        properties: {
+          __typename: { type: 'string' },
+          key: { type: 'string' },
+          value: { type: 'string' },
+        },
+        required: ['__typename', 'key', 'value'],
+      },
+    },
+    entityUrl: { $ref: 'EntityUrl' },
+    fieldAddress: {
+      type: 'object',
+      properties: {
+        locality: { type: 'string' },
+        administrativeArea: { type: 'string' },
+      },
+      required: ['locality', 'administrativeArea'],
+    },
+    fieldIntroText: { type: 'string' },
+    fieldOffice: { $ref: 'transformed/node-health_care_region_page' },
+    fieldPdfVersion: { $ref: 'Media' },
+    fieldPressReleaseContact: {
+      type: 'array',
+      maxItems: 1,
+      items: {
+        type: 'object',
+        properties: {
+          entity: {
+            type: 'object',
+            properties: {
+              title: { type: 'string' },
+              fieldDescription: { type: ['string', 'null'] },
+              fieldPhoneNumber: { type: 'string' },
+              fieldEmailAddress: { type: ['string', 'null'] },
+            },
+            required: [
+              'title',
+              'fieldDescription',
+              'fieldPhoneNumber',
+              'fieldEmailAddress',
+            ],
+          },
+        },
+        required: ['entity'],
+      },
+    },
+    fieldPressReleaseDownloads: { type: 'array' },
```

Comment:
I've never found this to be anything other than an empty array, so I'm not sure if it is actually used anymore.  According to the [template](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/layouts/press_release.drupal.liquid#L144-L164) the array items are objects with a wrapping `entity` property, and that entity has an `entityBundle` that can be one of:

- `document`
- `image`
- `video`

Knowing that, do we want this to be an array of `{ref: 'Media' }`?

Reply:
I'd leave it as
```
{
  type: 'array',
  maxItems: 0
}
```
for now and when that eventually throws an error, somebody can add the appropriate `$ref`. Just make sure to make a comment like "Only ever found an empty array; subject to change when this throws an error because it has real data."